data/json/monsters/slimes.json

```
@@ -174,8 +174,8 @@
   {
     "id": "mon_player_blob",
     "type": "MONSTER",
-    "name": { "str": "slimespring" },
-    "description": "A familiar-looking blob of goo.  It sprouts the occasional eyestalk.",
+    "name": { "str": "slime microbian" },
+    "description": "A small wriggling slime, designed as a first line of defense for the colony.",
```

Designed isn't really the right word, maybe just state what it's for.
```suggestion
    "description": "A small wriggling slime, a first line of defense for the colony.",
```